Packs/CyberArkIdentity/Integrations/CyberArkIdentityEventCollector/CyberArkIdentityEventCollector_description.md

```diff
@@ -5,16 +5,22 @@ Before you start, you need to get the tenant ID, app ID, username, and password.
 ## Configuration Parameters
 
 **Server URL**    
-The endpoint to get the logs. For example, ``https://{{tenant}}.my.idaptive.app/``.
+The endpoint to get the logs. For example, ``https://{{tenant}}.my.idaptive.app/``
 
 **App ID**  
 The application ID to fetch the logs from.
 
-**User name and Password**    
+**User name and Password** 
+
 The SIEM user name and password.
```

Comment:
I would change this line to:
The user that was created in CyberArk for XSIAM integration.